source/install.txt

```diff
@@ -10,168 +10,242 @@ Install Compass
-         executable file always installs |compass-short| to the
-         following path:
+     - id: windows
+       name: Windows
+       content: |
 
-         .. code-block:: none
+         To download |compass| to an Windows host using a web browser:
 
-            C:\Users\<user>\AppData\Local\<MongoDB Compass Edition>\
+         1. Open the {+download-page+}.
 
-         Once installed, |compass-short| launches automatically and prompts you to
-         configure privacy settings and specify update preferences.
+         #. Download the latest version of |compass| for Windows. The
+            |compass| installer is an ``.exe`` executable file.
 
      - id: ubuntu
        name: Ubuntu
        content: |
 
-         Download and Install from the MongoDB Downloads Page
-         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+         Download from the MongoDB Downloads Page
+         ````````````````````````````````````````
 
-         To install |compass| on Ubuntu, download the latest
-         |compass-short| package for Ubuntu from the {+download-page+}.
-         Double click on the ``.deb`` package to open the package using
-         Ubuntu Software. Click :guilabel:`Install`.
+         To download |compass| to an Ubuntu host using a web browser:
 
-         .. figure:: /images/compass/install-ubuntu.png
-            :figwidth: 654px
+         1. Open the {+download-page+}.
 
-         Once installed, launch Compass from your
-         :guilabel:`Applications` folder.
+         #. Download the latest version of |compass| for Ubuntu. The
+            |compass| installer is a ``.deb`` package.
 
-         Download and Install Using the Terminal
-         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+         Download using the Terminal
+         ```````````````````````````
 
-         You can also download and install |compass| via the terminal:
+         To download |compass| to an Ubuntu host using a shell:
 
-         #. Download the package:
+         1. Download the package using ``wget``:
 
             .. code-block:: sh
 
-               wget https://downloads.mongodb.com/compass/mongodb-compass_1.15.1_amd64.deb;
+               wget https://downloads.mongodb.com/compass/mongodb-compass_1.15.1_amd64.deb
 
-         #. Install Compass:
+            The |compass| installer is a ``.deb`` package.
 
-            .. code-block:: sh
+     - id: rhel
+       name: RHEL
+       content: |
 
-               sudo dpkg -i mongodb-compass_1.15.1_amd64.deb;
+         To download |compass| to a Red Hat Enterprise Linux host using
+         a shell:
 
-         #. Start Compass:
+         1. Download the package using ``wget``:
 
             .. code-block:: sh
 
-               mongodb-compass;
+               wget https://downloads.mongodb.com/compass/mongodb-compass-1.15.1.x86_64.rpm
 
-     - id: rhel
-       name: RHEL
+            The |compass| installer is an ``.rpm`` package.
+
+
+Install Compass
+---------------
+
+.. note::
```

Comment:
This note may be better moved up to the download section. Thoughts?

Reply:
+1